src/rpc/governance.cpp

```diff
@@ -950,7 +951,7 @@ static UniValue gobject_getcurrentvotes(const JSONRPCRequest& request)
     if (!request.params[1].isNull() && !request.params[2].isNull()) {
         uint256 txid = ParseHashV(request.params[1], "Masternode Collateral hash");
         std::string strVout = request.params[2].get_str();
-        mnCollateralOutpoint = COutPoint(txid, (uint32_t)atoi(strVout));
+        mnCollateralOutpoint = COutPoint(txid, (uint32_t)LocaleIndependentAtoi<int>(strVout));
```

Comment:
this code looks strange for me. Parse a number like a signed number and convert after that to unsigned? Is it what expected? @PastaPastaPasta

Reply:
`atoi` substitution was primarily scripted but will change template typedef to `uint32_t`